frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -1115,12 +1118,16 @@ export const CreateOperandYAML: React.FC<CreateOperandYAMLProps> = (props) => {
     // eslint-disable-next-line no-console
     console.error('Error parsing example JSON from annotation. Falling back to default.');
   }
-  const resourceObjPath = () =>
-    `${resourcePathFromModel(
-      ClusterServiceVersionModel,
-      props.match.params.appName,
-      props.match.params.ns,
-    )}/${props.match.params.plural}`;
+  const resourceObjPath = () => {
+    const params = new URLSearchParams(window.location.href);
+    return params.has('redirect')
```

Comment:
This doesn't work for operator backed services when I tested it. For some reason the `redirect` param that is added is being removed on this page which results in user getting redirected to default admin URL.

![Peek 2020-02-29 01-54](https://user-images.githubusercontent.com/6041994/75585083-515f8f00-5a97-11ea-85b0-ffd777305afc.gif)

Reply:
@rohitkrai03 Not sure why the param is being removed but I changed the implementation to use activePerspective.